pkg/controller/dynamodb/table/hooks.go

```diff
@@ -302,59 +318,55 @@ type updateClient struct {
 	client svcsdkapi.DynamoDBAPI
 }
 
-// nolint: gocyclo
-func (e *updateClient) preUpdate(_ context.Context, cr *svcapitypes.Table, u *svcsdk.UpdateTableInput) error {
-	switch aws.StringValue(cr.Status.AtProvider.TableStatus) {
-	case string(svcapitypes.TableStatus_SDK_UPDATING), string(svcapitypes.TableStatus_SDK_CREATING):
-		return nil
-	}
-	t, err := e.client.DescribeTable(&svcsdk.DescribeTableInput{TableName: aws.String(meta.GetExternalName(cr))})
+func (e *updateClient) preUpdate(ctx context.Context, cr *svcapitypes.Table, u *svcsdk.UpdateTableInput) error {
+
+	filtered := &svcsdk.UpdateTableInput{TableName: aws.String(meta.GetExternalName(cr))}
+
+	// The AWS API requires us to do one kind of update at a time per
```

Comment:
that's kind of crazy that the AWS API behaves this way - I hope we don't see that too often because this logic is quite manual and probably difficult to generate :)